data/languages/vuupper.sinc

```diff
@@ -0,0 +1,294 @@
+# VU upper instructions
```

Comment:
What about bits (26, 31)

instead of is vuco=0b1, what about something like

MACRO:  is prime=0b010010 & vuco=0b1 { export 0:1; }

and then using is MACRO for a vu macro instruction